test/tests_for_sdkmr/EE2Status_test.py

```diff
@@ -19,9 +19,10 @@
 def _finish_job_complete_minimal_get_test_job(job_id, sched, app_id, gitcommit, user):
     job = Job()
     job.id = ObjectId(job_id)
-    job.running = 123.0
-    job.finished = 456.5
+
+    job.finished = job.id.generation_time.timestamp() + 10
     job.status = Status.running.value
```

Comment:
If the job is finished how can the status be running?

Reply:
Looks like this was my fault based on git blame. Leave it as is for now as it might break other things

Reply:
Well, maybe I was refactoring older code, but still, it's been like this for 2+ years